Sources/Views/ALKMessageBaseCell.swift

```diff
@@ -168,7 +178,8 @@ open class ALKMessageCell: ALKChatBaseCell<ALKMessageViewModel> {
         mentionStyle: Style
     ) {
         self.viewModel = viewModel
-
+        iframeView.isHidden = true
+        iframeWidth.isActive = false
```

Comment:
can't we move it inside the initialisation of iframeview similar to is opaque

Reply:
its need to be there as iframeView is initialised only one time and when ever there is some iframe msg comes that iframeView will be set to not hidden state that can effect when there is non iframe msg comes up.
For iFrameWidth that is the constraint that also needed to be there as update will handle that it's active or not.